lib/minify.js

```diff
@@ -10,6 +10,10 @@ exports.simple_glob = function(files) {
     return files;
 };
 
+var defaultBase64Decoder = exports.base64Decoder = function(input) {
```

Comment:
Not keen on the `r` at the end of the function name. Would suggest one of the following: `base64decode`, `base64_decode`, or `base64Decode`.